redbot/cogs/general/general.py

```diff
@@ -234,38 +240,210 @@ def __init__(self):
+                "us-central": _("US Central ") + "\U0001F1FA\U0001F1F8",
+                "singapore": _("Singapore ") + "\U0001F1F8\U0001F1EC",
+                "sydney": _("Sydney ") + "\U0001F1E6\U0001F1FA",
+                "brazil": _("Brazil ") + "\U0001F1E7\U0001F1F7",
+                "hongkong": _("Hong Kong ") + "\U0001F1ED\U0001F1F0",
+                "russia": _("Russia ") + "\U0001F1F7\U0001F1FA",
+                "japan": _("Japan ") + "\U0001F1EF\U0001F1F5",
+                "southafrica": _("South Africa ") + "\U0001F1FF\U0001F1E6",
+                "india": _("India ") + "\U0001F1EE\U0001F1F3",
+                "dubai": _("Dubai ") + "\U0001F1E6\U0001F1EA",
+                "south-korea": _("South Korea ") + "\U0001f1f0\U0001f1f7",
+            }
+            verif = {
+                "none": _("0 - None"),
+                "low": _("1 - Low"),
+                "medium": _("2 - Medium"),
+                "high": _("3 - Hard"),
+                "extreme": _("4 - Extreme"),
+            }
+
+            features = {
+                "PARTNERED": _("Partnered"),
+                "VERIFIED": _("Verified"),
+                "DISCOVERABLE": _("Server Discovery"),
+                "PUBLIC": _("Public"),
+                "INVITE_SPLASH": _("Splash Invite"),
+                "VIP_REGIONS": _("VIP Voice Servers"),
+                "VANITY_URL": _("Vanity URL"),
+                "MORE_EMOJI": _("More Emojis"),
+                "COMMERCE": _("Commerce"),
+                "LURKABLE": _("Lurkable"),
+                "NEWS": _("News Channels"),
+                "ANIMATED_ICON": _("Animated Icon"),
+                "BANNER": _("Banner Image"),
+                "MEMBER_LIST_DISABLED": _("Member list disabled"),
+            }
+            guild_features_list = [
+                f"✅ {name}" for feature, name in features.items() if feature in set(guild.features)
+            ]
+
+            joined_on = _(
+                "{bot_name} joined this server on {bot_join}. That's over {since_join} days ago!"
+            ).format(
+                bot_name=ctx.bot.user.name,
+                bot_join=guild.me.joined_at.strftime("%d %b %Y %H:%M:%S"),
+                since_join=humanize_number((ctx.message.created_at - guild.me.joined_at).days),
+            )
+
+            data = discord.Embed(description=created_at, colour=await ctx.embed_colour())
+            data.set_author(
+                name=guild.name,
+                icon_url="https://cdn.discordapp.com/emojis/457879292152381443.png"
+                if "VERIFIED" in guild.features
+                else "https://cdn.discordapp.com/emojis/508929941610430464.png"
+                if "PARTNERED" in guild.features
+                else discord.Embed.Empty,
+            )
+            data.set_thumbnail(
+                url=guild.icon_url
+                if guild.icon_url
+                else "https://cdn.discordapp.com/embed/avatars/1.png"
+            )
```

Comment:
I think it's better to just not show server icon at all if it's not set instead of using unrelated one.
```suggestion
            if guild.icon_url:
                data.set_thumbnail(url=guild.icon_url)
```